new-system-tests/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>apicurio-registry</artifactId>
         <groupId>io.apicurio</groupId>
-        <version>2.2.6-SNAPSHOT</version>
+        <version>2.3.2-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
```

Comment:
I think we should remove the "new" word from the module name. These are the system tests written in java, we never had something like this.

Reply:
Yes, I will remove word "new" from the module name when we will have some accurate test coverage, because there is still "old" version of Java test suite under `system-tests` directory.